commcare/src/main/resources/velocity.templates/task-data-provider.vm

```diff
@@ -163,7 +163,7 @@
                                     "displayName" : "commcare.case.field.caseName",
                                     "fieldKey" : "caseName"
                                 },
-                                #foreach($caseField in $entry.value)
+                                #foreach($caseField in $entry.getApplicationName())
```

Comment:
I was referring to "caseProperties" field. caseType is a name of the Case and caseProperties are fields of that case.